llvm/test/MC/AArch64/armv9.6a-lsui.s

```diff
@@ -0,0 +1,395 @@
+# NOTE: Assertions have been autogenerated by utils/update_mca_test_checks.py
+// RUN: llvm-mc -triple aarch64 -mattr=+lsui -show-encoding %s  | FileCheck %s
```

Comment:
These tests should also check that the instructions are rejected when the subtarget feature is not enabled.

Reply:
Done